Update Understanding-ROS2-Topics.rst (#4803)
ros2/ros2_documentation: source/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics.rst

Recall that you set the rate of ``turtle1/cmd_vel`` to publish at a steady 1 Hz using ``ros2 topic pub --rate 1``.
If you run the above command with ``turtle1/cmd_vel`` instead of ``turtle1/pose``, you will see an average reflecting that rate.

9 ros2 topic bw
^^^^^^^^^^^^^^^

The bandwidth used by a topic can be viewed using:

.. code-block:: console

    ros2 topic bw /turtle1/pose

It returns the bandwidth utilization and number of messages being published ``pose`` on the topic.

.. code-block:: console

    Subscribed to [/turtle1/pose]
    1.51 KB/s from 62 messages
    	Message size mean: 0.02 KB min: 0.02 KB max: 0.02 KB

The number of messages can be verified by running the ``echo`` command. 

> If you run the above command with ``turtle1/cmd_vel`` instead of ``turtle1/pose``, the number of messages should be 2.

Reviewer: I do not really understand this sentence, can you rephrase this? does this mean ros2 topic echo command has the ability to count the message? (that i do not think it does) besides, `bw` sub-command already shows the number of messages in the output above?

Author: I was trying to say running the ``echo`` command let's you view the messages and count them to verify the number of messages. But I'll word it better

10 ros2 topic find

> ^^^^^^^^^^^^^^^

Reviewer: ```suggestion
10 ros2 topic find
^^^^^^^^^^^^^^^^^^
```

To list a list of available topics of a given type use:

.. code-block:: console

    ros2 topic find <topic_type>

Recall that the ``cmd_vel`` topic has the type:

.. code-block:: console

    geometry_msgs/msg/Twist

Using the ``find`` command outputs topics available when given the message type:

.. code-block:: console

    ros2 topic find geometry_msgs/msg/Twist

This outputs:

.. code-block:: console

    /turtle1/cmd_vel

.. 10 rqt_plot

>    ^^^^^^^^^^

Reviewer: ```suggestion
.. 11 rqt_plot
   ^^^^^^^^^^^
```

   Can't do this section now because there's some significant UI issues with rqt_plot for ROS 2

11 Clean up

> ^^^^^^^^^^

Reviewer: ```suggestion
11 Clean up
^^^^^^^^^^^
```

Recall that you set the rate of ``turtle1/cmd_vel`` to publish at a steady 1 Hz using ``ros2 topic pub --rate 1``.
If you run the above command with ``turtle1/cmd_vel`` instead of ``turtle1/pose``, you will see an average reflecting that rate.

9 ros2 topic bw
^^^^^^^^^^^^^^^

The bandwidth used by a topic can be viewed using:

.. code-block:: console

    ros2 topic bw /turtle1/pose

> It returns the bandwidth utilization and number of messages being published ``pose`` on the topic.

Reviewer 2: ```suggestion
It returns the bandwidth utilization and number of messages being published to the ``/turtle1/pose`` topic.
```

.. code-block:: console

    Subscribed to [/turtle1/pose]
    1.51 KB/s from 62 messages
        Message size mean: 0.02 KB min: 0.02 KB max: 0.02 KB

10 ros2 topic find
^^^^^^^^^^^^^^^^^^

To list a list of available topics of a given type use:

.. code-block:: console

    ros2 topic find <topic_type>

Recall that the ``cmd_vel`` topic has the type:

.. code-block:: console

    geometry_msgs/msg/Twist

Using the ``find`` command outputs topics available when given the message type:

.. code-block:: console

    ros2 topic find geometry_msgs/msg/Twist

This outputs:

.. code-block:: console

    /turtle1/cmd_vel

> .. 10 rqt_plot
>    ^^^^^^^^^^^
>    Can't do this section now because there's some significant UI issues with rqt_plot for ROS 2

Reviewer 2: Let's just remove this, it doesn't belong in this tutorial anyway:

```suggestion
```

Recall that you set the rate of ``turtle1/cmd_vel`` to publish at a steady 1 Hz using ``ros2 topic pub --rate 1``.
If you run the above command with ``turtle1/cmd_vel`` instead of ``turtle1/pose``, you will see an average reflecting that rate.

9 ros2 topic bw
^^^^^^^^^^^^^^^

The bandwidth used by a topic can be viewed using:

.. code-block:: console

    ros2 topic bw /turtle1/pose

It returns the bandwidth utilization and number of messages being published ``pose`` on the topic.

.. code-block:: console

    Subscribed to [/turtle1/pose]
    1.51 KB/s from 62 messages
    	Message size mean: 0.02 KB min: 0.02 KB max: 0.02 KB

The number of messages in the output above can be verified by running the ``echo`` command and manually counting the number of messages in the output. 

> If you run the ``bw`` command with ``turtle1/cmd_vel`` instead of ``turtle1/pose``, the number of messages should be 2.

Reviewer: Do we need this? I would remove these lines and keep the beginner tutorial to be simple for introduction about what we can do with those commands. what do you think?

Author: I added it because I thought it gave more clarity. But it's alright without it. I'll make the changes
